guides/common/modules/proc_adding-a-passphrase-to-ssh-key-used-for-remote-execution.adoc

```diff
@@ -0,0 +1,16 @@
+[id="adding-a-passphrase-to-ssh-key-used-for-remote-execution_{context}"]
+= Adding a passphrase to SSH key used for remote execution
+
+By default, {SmartProxy} uses a non-passphrase protected SSH key to execute remote jobs on hosts.
+You can protect the SSH key with a passphrase by following this procedure.
+
+.Procedure
+* On your {ProjectServer} or {SmartProxyServer}, use `ssh-keygen` to add a passphrase to your SSH key:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# ssh-keygen -p -f ~foreman-proxy/.ssh/id_rsa_foreman_proxy
```

Comment:
Should this mention that this is just the default path?

Reply:
I was considering something like "if you changed the location of your ssh key, then use the new location" but just sounds too obvious, so I would probably opt for not mentioning that unless there are objections.